### PR TITLE
Url test

### PR DESCRIPTION
**Pull request by xeon-zolt**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes (provide a screenshot or link for test) 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
it will show the status of the build by checking the url status

#### Changes proposed in this pull request:

- addded javascript to keep checking the status of the link

Fixes #
#27 ,#29(REMOVED YIELD)

### CODE REVIEW

**Comment by xeon-zolt** [comment]
#63

**Comment by meets2tarun** [comment]
@xeon-zolt Please provide a test-URL of your made changes, so that it would be easy for the one to review.
Thanks

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 language: python
+python:
+  - "3.6"
```
Comment (abishekvashok) [inline .travis.yml]: No need of quotes

**Comment by xeon-zolt** [comment]
https://meilix-generator-test.herokuapp.com/

**Comment by abishekvashok** [comment]
Pls check Travis and squash the commits.

**Pull request closed without merging**